src/components/MyPage.js

```diff
@@ -77,6 +58,49 @@ class MyPage extends Component {
     UserUtil.getRequestSetting(this.onRequestCallback.bind(this));
   }
 
+  getProfileImage(status) {
+    let image;
+    if (status.profile_picture) {
+      image = { uri: status.profile_picture };
+      return image;
+    } else {
+      image = require('../resources/pattern.png');
+      return image;
+    }
+  }
+
+  getCurrentStatus(status) {
+    let currentTask;
+    let location;
+
+    if (status.experience.length > 0) {
+      location = status.experience[0].employer.name;
+      if (status.experience[0].position) {
+        currentTask = status.experience[0].position.name;
+      } else {
+        return location;
+      }
+
+      return currentTask + ' at ' + location;
+    }  else if (status.education.length > 0) {
+      let lastIndex = status.education.length - 1;
+      let education = status.education[lastIndex];
```

Comment:
It is good to use 'const'.
let lastIndex -> const lastIndex
let education -> const lastIndex

You can refer this URL
http://eslint.org/docs/rules/prefer-const.html